src/main/java/com/google/step/servlets/CommentServlet.java

```diff
@@ -0,0 +1,79 @@
+package com.google.step.servlets;
+
+import com.google.step.datamanager.CommentManager;
+import com.google.step.datamanager.CommentManagerDatastore;
+import com.google.step.model.Comment;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles individual comments. */
+@WebServlet("/api/comments/*")
+public class CommentServlet extends HttpServlet {
+
+  private CommentManager manager = new CommentManagerDatastore();
+
+  public CommentServlet(CommentManager commentManager) {
+    manager = commentManager;
+  }
+
+  /** Deletes the comment with the given id parameter */
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    long id;
+    try {
+      id = Long.parseLong(request.getPathInfo().substring(1));
+    } catch (NumberFormatException e) {
+      response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+      return;
+    } catch (StringIndexOutOfBoundsException e) {
+      response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+      return;
+    }
```

Comment:
can catch both exceptions in just one catch statement

Reply:
i removed the second catch instead cos i realised it wont be called